h2o-docs/src/product/cloud-integration/aws.rst

```diff
@@ -1,54 +1,6 @@
 Using H2O with Amazon AWS
 ~~~~~~~~~~~~~~~~~~~~~~~~~
 
-AWS Marketplace provides a straightforward way to deploy a cluster of VMs with H2O.
+The old `H2O Artificial Intelligence instance <https://aws.amazon.com/marketplace/pp/prodview-76ewxkmzkvr3m>`_ is now unavailable.
```

Comment:
One small change: "...is now unavailable." sounds clunky. Please change it to: "...is no longer available."

Reply:
great edit, thanks. done